README.md

```diff
@@ -1,303 +1,388 @@
-## Getting Started  
-It is intended and recommended that InSpec and this profile be run from a __"runner"__ host (such as a DevOps orchestration server, an administrative management system, or a developer's workstation/laptop) against the target.
-
-__For the best security of the runner, always install on the runner the _latest version_ of InSpec and supporting Ruby language components.__
-
-Latest versions and installation options are available at the [InSpec](http://inspec.io/) site.
+# Microsoft 365 Foundation CIS Benchmark
+This InSpec Profile was created to facilitate testing and auditing of `CIS Microsoft 365 Benchmark`
+infrastructure and applications when validating compliancy with [Center for Internet Security (CIS) Benchmark](https://www.cisecurity.org/cis-benchmarks)
+requirements.
+ 
+- Profile Version: **3.1.0**
```

Comment:
This would technically be a patch update of this semver. It's still a profile representing v3.1 of the underlying benchmark, but we have updated it.

So this should be 3.1.1 both here and in the inspec.yml.